### ISSUE TITLE
Problem: a even_symmetric structure with odd_symmetric source

### ISSUE THREAD

**Issue by chenshang2** [NanoComp/meep#2664]
Hello everyone.

I have some problems when indicating the symmetries in a simulation. so for a structure with mirror symmetry along y plane, and pumped by an odd source (TE modes), you need a -1 phase factor to calculate the right field. I am simulating this kind of device and I find that when I indicate: mp.Mirror(mp.Y,phase=-1) with a TE0 mode. the simulation will not converge and keep running, the DFT field shows that in the beginning the field seems fine but after around 200 time steps the field just disappears. Do you have any suggestions like where I should be more careful? Thanks!

**Comment by stevengj** [comment]
> the DFT field shows that in the beginning the field seems fine but after around 200 time steps the field just disappears. 

Have never seen that.  Is it blowing up or something?

**Comment by chenshang2** [comment]
> > the DFT field shows that in the beginning the field seems fine but after around 200 time steps the field just disappears.
> 
> Have never seen that. Is it blowing up or something?

Yeah I tried v1.22.0, and in a low resolution (10pixel/um), the simulation blows up with an error that the simulated field goes to inf..... but right now the simulation is fine after I double the resolution. Just curious why the symmetry could blow up the simulation in my previous case? I used to think that the symmetry only tried to copy the field and with certain phase factors.
